### PR TITLE
feat / send tg message

### PR DESCRIPTION
**Pull request by irondsd**
est: 1h

### CODE REVIEW

**Comment by irondsd** [comment]
Waiting for the vercel project to be assigned to make sure it's all working

File: api/send.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import axios from 'axios' 
+
+export default async function (request: VercelRequest, response: VercelResponse) {
```
Comment (georgeciubotaru) [inline api/send.ts:4]: @irondsd What if the message will be larger, better to use POST request a think.
What do you say?

Comment (irondsd) [reply api/send.ts:4]: You mean post request with body?

Comment (georgeciubotaru) [reply api/send.ts:4]: yes

**Pull request merged**